odoo/http.py

```diff
@@ -2859,9 +2852,7 @@ def fake_start_response(status, headers):
             except Exception as exc:
                 # Valid (2xx/3xx) response returned via werkzeug.exceptions.abort.
                 if isinstance(exc, HTTPException) and exc.code is None:
-                    response = exc.get_response()
-                    HttpDispatcher(request).post_dispatch(response)
-                    return response(environ, start_response)
+                    return exc.get_response()(environ, start_response)
```

Comment:
wrong is there are situations where we use `abord()` in `_serve_nodb`, maybe we should do like `_serve_db` and `try: except HTTPException if code is None: post_dispatch(exc.get_resposne()); raise`